### ISSUE TITLE
unable to install on arch linux

### ISSUE THREAD

**Issue by SamDc73** [marianosimone/epub-thumbnailer#37]
the message error :
```
Checking dependencies ...

Required dependencies:
    Python Imaging Library .......
OK

Installing epub-thumbnailer to /usr/bin/ ...
Could not install
```
i'm running python version 3.10.4

i have pillow installed on my machine.

**Comment by marianosimone-stripe** [comment]
Given the sequence of log messages, it seems like it failed in this line:

https://github.com/marianosimone/epub-thumbnailer/blob/486111b27d192c27011229dc696e9179f59b0eab/install.py#L154-L156

Coming from this  `if`:

https://github.com/marianosimone/epub-thumbnailer/blob/486111b27d192c27011229dc696e9179f59b0eab/install.py#L124-L125

Did you run the installer as `sudo`? This seems to be a permissions error to me

**Comment by marianosimone** [comment]
Hi @SamDc73, thanks  for reporting this.

Given the sequence of log messages, it seems like it failed in this line:

https://github.com/marianosimone/epub-thumbnailer/blob/486111b27d192c27011229dc696e9179f59b0eab/install.py#L154-L156

Coming from this  `if`:

https://github.com/marianosimone/epub-thumbnailer/blob/486111b27d192c27011229dc696e9179f59b0eab/install.py#L124-L125

Did you run the installer as `sudo`? This seems to be a permissions error to me

**Comment by SamDc73** [comment]
> Did you run the installer as sudo? This seems to be a permissions error to me

yes, i tried installing it using this command  `sudo python3 install.py install`

Are there any way i can installed locally (in the .local/bin directory) ?
